src/catalogue.ts

```diff
@@ -114,6 +142,21 @@ export class CatalogueModule {
         });
     }
 
+    public async loadItemDescriptionObjects(
+        projectId: string,
+        mdObj: any,
+        attributesMap: any,
```

Comment:
I'm not sure how hard it is but... please try to specify a type for arguments... at least for this new public interface...